python/hsfs/core/feature_group_engine.py

```diff
@@ -154,17 +148,6 @@ def commit_delete(feature_group, delete_df, write_options):
         )
         return hudi_engine_instance.delete_record(delete_df, write_options)
 
-    def update_config(self, feature_group, metadata):
```

Comment:
I think this function was old and not actually used anymore, so you shouldn't need it in the base class. but please double check.

Reply:
fixed